mypyc/genops.py

```diff
@@ -867,56 +858,81 @@ def visit_conditional_expr(self, expr: ConditionalExpr) -> Register:
 
         return target
 
-    def translate_special_method_call(self, callee: MemberExpr,
-                                      expr: CallExpr) -> Optional[Register]:
+    def translate_special_method_call(self,
+                                      base_reg: Register,
+                                      name: str,
+                                      args: List[Register],
+                                      result_type: Optional[RType],
+                                      line: int,
+                                      temp_result: bool = False) -> Optional[Register]:
```

Comment:
I like passing in `target: Register = INVALID_REGISTER` and then passing that to `alloc_temp` as the idiom for maybe putting in a temporary or not.